ntbs-service/Pages/Search/Index.cshtml

```diff
@@ -26,20 +26,48 @@
                     <span class="nhsuk-hint " id="example-hint">
                         Search by NTBS, ETS, LTBR or NHS number
                     </span>
-                    <span ref="errorField" nhs-span-type="ErrorMessage" id="id-error"
+                    <span nhs-span-type="ErrorMessage" id="id-error"
                         asp-validation-for="SearchParameters.IdFilter"></span>
                     <input is-error-input=@IdFilterError nhs-input-type="Standard"
                         asp-for="SearchParameters.IdFilter" classes="nhsuk-input--width-10" />
                 </nhs-form-group>
             </nhs-grid-column>
+            <nhs-grid-column grid-column-width="OneQuarter">
+                @{
+                    var FamilyNameError = !Model.validationService.IsValid("SearchParameters.FamilyName");
+                    var FamilyNameGroupState = FamilyNameError ? Error : Standard;
+                }
+                <nhs-form-group nhs-form-group-type=@FamilyNameGroupState>
+                    <label nhs-label-type="Standard"> Family Name</label>
+                    <span class="nhsuk-hint" id="example-hint">Enter at least two characters</span>
+                    <span nhs-span-type="ErrorMessage" id="id-error"
+                        asp-validation-for="SearchParameters.FamilyName"></span>
+                    <input is-error-input=@FamilyNameError nhs-input-type="Standard"

```

Comment:
Could we please add an `aria-describedby="example-hint id-error"` to link everything up?
- or equivalent names for the hint/error message which could potentially be a bit more descriptive if we're going to have several on the page (currently two elements with `id="id-error"`